src/plugins/intel_gna/src/memory/gna_mem_regions.hpp

```diff
@@ -7,6 +7,8 @@
 #include <cstdint>
 #include <vector>
 
+#include "log/debug.hpp"
```

Comment:
Is this include needed?

Reply:
No, for this PR, but it fixes compile error in debug